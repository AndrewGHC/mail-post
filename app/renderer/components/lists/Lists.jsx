// @flow
import React, { Fragment } from 'react';

import NewList from './new-list';

const Lists = () => (
  <Fragment>
    <NewList />
  </Fragment>
);

export default Lists;
