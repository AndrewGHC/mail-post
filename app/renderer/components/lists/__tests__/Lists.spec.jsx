import React from 'react';
import { shallow } from 'enzyme';

import Lists from '../Lists';

describe('Lists', () => {
  it('renders elements', async () => {
    const wrapper = shallow(<Lists />);
    expect(wrapper).toMatchSnapshot();
  });
});
