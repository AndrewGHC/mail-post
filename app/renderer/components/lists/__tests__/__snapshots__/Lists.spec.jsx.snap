// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Lists matches snapshot 1`] = `
<React.Fragment>
  <NewList />
</React.Fragment>
`;
